tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,116 +1,83 @@
+      btcTxHelper = getBtcClient();
+
+      if(process.env.RUNNING_SINGLE_TEST_FILE) {
+          await fulfillRequirementsToRunAsSingleTestFile();
+      };
+  });
+
+
+  it('should do a multiple pegouts from the same rsk address after making a pegin', async () => {
+    const btcAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
+    await whitelistingAssertions.assertAddLimitedLockWhitelistAddress(rskTxHelper, btcAddressInfo.address, btcEthUnitConverter.btcToSatoshis(PEGIN_VALUE_IN_BTC));
+    await rskUtils.mineAndSync(rskTxHelpers);
+    
+    const recipientRskAddressInfo = getDerivedRSKAddressInformation(btcAddressInfo.privateKey, btcTxHelper.btcConfig.network);
+    await rskTxHelper.importAccount(recipientRskAddressInfo.privateKey);
+    const unlocked = await rskTxHelper.unlockAccount(recipientRskAddressInfo.address);
+    expect(unlocked, 'Account was not unlocked').to.be.true;
+
+    const latestActiveForkName = await getLatestActiveForkName();
+    const bridge = getBridge(rskTxHelper.getClient(), latestActiveForkName);
+    const federationAddress = await bridge.methods.getFederationAddress().call();
+
+    const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
+
+    await btcTxHelper.fundAddress(btcAddressInfo.address, PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
     
-    // Mine a few rsk blocks to prevent being at the beginning of the chain,
-    // which could trigger border cases we're not interested in
-    await sequentialPromise(10, () => rskUtils.mineAndSync(rskTxHelpers));
+    const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, PEGIN_VALUE_IN_BTC);
+    await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+    const recipientRskAddressBalanceAfterPegin = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+    expect(Number(recipientRskAddressBalanceAfterPegin)).to.be.equal(btcEthUnitConverter.btcToWeis(PEGIN_VALUE_IN_BTC));
+
+    await sendTxToBridge(rskTxHelper, PEGOUT_VALUE_IN_RBTC, recipientRskAddressInfo.address);
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
+
+    const federationAddressBalanceAfterFirstPegout = Number(await btcTxHelper.getAddressBalance(federationAddress));
+    expect(Number(federationAddressBalanceAfterFirstPegout)).to.be.equal(Number(federationAddressBalanceInitial + 2 * MINIMUM_PEGIN_VALUE_IN_BTC));
+
+    const senderAddressBalanceAfterFirstPegout = Number(await btcTxHelper.getAddressBalance(btcAddressInfo.address));
+    expect(Number(senderAddressBalanceAfterFirstPegout)).to.be.above(MINIMUM_PEGIN_VALUE_IN_BTC - btcTxHelper.getFee()).and.below(MINIMUM_PEGIN_VALUE_IN_BTC);
+
+    const recipientRskAddressAfterFirstPegout = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+    expect(Number(recipientRskAddressAfterFirstPegout)).to.be.above(PEGOUT_VALUE_IN_WEIS - RSK_TX_FEE_IN_RBTC).and.below(2 * PEGOUT_VALUE_IN_WEIS);
```

Comment:
```suggestion
  expect(Number(recipientRskAddressAfterFirstPegout)).to.be.above(btcToWeis(PEGIN_VALUE_IN_BTC - PEGOUT_VALUE_IN_RBTC - RSK_TX_FEE_IN_RBTC)).and.below(btcToWeis(PEGIN_VALUE_IN_BTC - PEGOUT_VALUE_IN_RBTC));
```
Be careful with the units, you are doing `PEGOUT_VALUE_IN_WEIS - RSK_TX_FEE_IN_RBTC` here. Weis - RBTC